addons/sale_timesheet/tests/test_sale_timesheet_portal.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+from odoo import Command
+from odoo.tests import tagged
+
+from odoo.addons.project.tests.test_project_sharing import TestProjectSharingCommon
+from odoo.addons.sale_timesheet.tests.common import TestCommonSaleTimesheet
+
+@tagged('post_install', '-at_install')
+class TestPortalSaleTimesheet(TestProjectSharingCommon, TestCommonSaleTimesheet):
```

Comment:
Small nitpick, but it would be slightly better if the class had the same name as the file